docs/supported-versions.asciidoc

```diff
@@ -1,7 +1,7 @@
 * link:https://kubernetes.io/docs/tasks/tools/install-kubectl/[kubectl] 1.11+
 * Kubernetes 1.12+ or OpenShift 3.11+
 * Google Kubernetes Engine (GKE), Azure Kubernetes Service (AKS), and Amazon Elastic Kubernetes Service (EKS)
-* Elastic Stack: 6.8+, 7.1+
+* Elasticsearch, Kibana, APM Server, Enterprise Search: 6.8+, 7.1+
```

Comment:
I don't think there is an Enterprise Search in 6.8. I believe the first version we support is 7.6